wcivf/apps/core/context_processors.py

```diff
@@ -2,9 +2,15 @@
 
 from .forms import PostcodeLookupForm
 
+# Remove this once we add all_settings to settings.CONTEXT_PROCESSORS
 def canonical_url(request):
     return {'CANONICAL_URL': settings.CANONICAL_URL}
 
+
+def all_settings(request):
+    return dict(settings={x: getattr(settings, x) for x in dir(settings)})
```

Comment:
I reckon an `if not x.startswith('_')` would be good here – we don't want python's magic `__whatever__` vars in the context I don't think.